draft-ietf-oauth-status-list.md

```diff
@@ -192,12 +192,28 @@ This section defines the structure for a JSON-encoded Status List:
 The following example illustrates the JSON representation of the Status List:
 
 ~~~~~~~~~~
-{::include ./examples/status_list_encoding}
+{::include ./examples/status_list_encoding_json}
 ~~~~~~~~~~
 
 ## Status List in CBOR Format {#status-list-cbor}
 
-TBD
+This section defines the `StatusList` structure for a CBOR-encoded Status List.
+
+The `StatusList` structure is a map (Major Type 5) and defines the following entries:
```

Comment:
Are we considering to register a CBOR tag for StatusList itself?

Reply:
No, this is not needed.

Reply:
Why not? The StatusList is a separate data structure that may live on its own, dependent from the StatusListToken. Doesn't it make sense in this case or should this be solved by media types alone?

Reply:
Tags are usually not needed. In all cases, the structure of the CBOR is pre-defined. That is why CBOR tags are not really needed in most cases.